src/components/home-comps/features/index.module.less

```
@@ -35,41 +35,9 @@
-        }
-      }
-
-      .windows-icon {
-        width: 18px;
-        height: 18px;
-        path {
-          fill: var(--home-blog-btn-color) !important;
-        }
+      .platform-icon {
+        --icon-size: 18px;
+        color: var(--home-blog-btn-color);
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)